frame/support/procedural/src/storage/storage_struct.rs

```diff
@@ -86,7 +86,10 @@ pub fn decl_and_impl(scrate: &TokenStream, def: &DeclStorageDefExt) -> TokenStre
 			Ident::new(INHERENT_INSTANCE_NAME, Span::call_site())
 		};
 
-		let storage_name_str = syn::LitStr::new(&line.name.to_string(), line.name.span());
+		let storage_name_bstr = syn::LitByteStr::new(
+			&line.name.to_string().into_bytes()[..],
```

Comment:
```suggestion
			&line.name.to_string(),
```

`String` implements `AsRef<[u8]>`. Shouldn't that be enough?

Reply:
ah yes using `line.name.to_string().as_ref(),` done